src/Raven.Server/Config/Categories/ConfigurationCategoryType.cs

```diff
@@ -38,6 +38,7 @@ public enum ConfigurationCategoryType
         [Description("Traffic Watch")]
         TrafficWatch,
         [Description("Export & Import")]
-        ExportImport
+        ExportImport,
+        Etw
```

Comment:
how about putting that into Debug to have keys like `Debug.EventListener.Mode` ?

Reply:
done